packages/orderbook/src/seaport/seaport-lib-factory.ts

```diff
@@ -48,11 +48,9 @@ export class SeaportLibFactory {
   ) { }
 
   create(orderSeaportVersion?: SeaportVersion, orderSeaportAddress?: string): SeaportLib {
-    const seaportVersion = orderSeaportVersion ?? SEAPORT_CONTRACT_VERSION_V1_5;
     const seaportContractAddress = orderSeaportAddress ?? this.defaultSeaportContractAddress;
 
     return new SeaportLib(convertToV6Provider(this.provider), {
-      seaportVersion,
```

Comment:
seaport-js v4 only supports 1.5 and 1.6, which have identical ABIs, so the version is no longer passed in when initializing the lib